src/components/Video/Intro.js

```diff
@@ -28,7 +28,7 @@ const Intro = ({ translate, parseTranslatedTags, nextStep }: Props) => (
           <li key={key} className={style.introBullet}>
             <span className={classNames(style.introIcon, style[`${key}Icon`])} />
             { parseTranslatedTags(`capture.liveness.intro.${key}`, ({ text }) => (
-               <span className={style.bolder}>{text}</span>
+                <span className={style.bolder}>{text}</span>
```

Comment:
@kopijunkie Would you like to add a tech debt to add proper linting to this project? I see such all the time here...

Reply:
yes, that's a great suggestion!

Reply:
Just saw that there's already a ticket created for this in JIRA. 😄